css/portal-dashboard/feedback/feedback-info.less

```diff
@@ -0,0 +1,99 @@
+@import "../variables";
+
+.feedbackInfo {
+  background: @feedback-green-light5;
+
+  .titleWrapper {
+    align-items: center;
+    background: @feedback-green-light4;
+    border-bottom: solid 1px @feedback-green;
+    display: flex;
+    flex: 0 0 50px;
+    height: 50px;
+    text-align: center;
+    user-select: none;
+    width: 100%;
+  
+    .title {
+      margin: 15px 10px 16px 24px;
+    }
+  
+    .viewToggle {
+  
+      button {
+        background-color: @feedback-green-light4;
+        border: solid 1px @feedback-green;
+        cursor: pointer;
+        font-size: 100%;
+
+        &:hover {
+          background-color: @feedback-green-light3;
+        }
+        &:before {
+          background-color: @feedback-green
+        }
+        &.active {
+          background-color: @feedback-green;
+          color: #fff;
+
+          &:hover {
+            background-color: @feedback-green;
+          }
+          &:before {
+            background-color: #fff;
+          }
+        }
+  
+        &.viewToggle__button__activity {
```

Comment:
I know it's a can of worms, but is this CSS class naming convention that uses underscores used elsewhere in the project?  I know we've leaned towards pure camelCase in the projects that I've worked on that use CSS vars.  That said, I've only worked on a couple so maybe this practice is common in our other projects?